evalml/pipelines/components/component_base.py

```diff
@@ -273,10 +275,17 @@ def _handle_nullable_types(self, X=None, y=None):
         y_bool_incompatible = "y" in self._boolean_nullable_incompatibilities
         y_int_incompatible = "y" in self._integer_nullable_incompatibilities
         if y is not None and (y_bool_incompatible or y_int_incompatible):
-            y = _downcast_nullable_y(
-                y,
-                handle_boolean_nullable=y_bool_incompatible,
-                handle_integer_nullable=y_int_incompatible,
-            )
-
+            if isinstance(y, pd.Series):
+                y = _downcast_nullable_y(
+                    y,
+                    handle_boolean_nullable=y_bool_incompatible,
+                    handle_integer_nullable=y_int_incompatible,
+                )
+            # if y is a dataframe (from unstacked multiseries) use _downcast_nullable_X since downcast_nullable_y is for series
```

Comment:
Would it be possible to update `_downcast_nullable_y` to handle the multiseries case, rather than needing to call `_downcast_nullable_X`?